src/main/java/seedu/address/model/person/predicate/ContactStatusMatchPredicate.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.person.predicate;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CONTACTED_DATE;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.function.Predicate;
+
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.parser.ArgumentMultimap;
+import seedu.address.model.person.ContactedDate;
+import seedu.address.model.person.Person;
+
+/**
+ * A class that encapsulates the logic of contacted status check for a {@link Person}.
+ */
+public class ContactStatusMatchPredicate implements Predicate<Person> {
```

Comment:
The person data field is named as `ContactedDate`, would be nice if the naming is consistent, maybe `ContactedDateMatchPredicate`? 😄